ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -132,9 +133,7 @@ function SignatureIntegrationForm({
     initialValues = null,
     isEditable = false,
 }: IntegrationFormProps<SignatureIntegration>): ReactElement {
-    const formInitialValues = initialValues
-        ? ({ ...defaultValues, ...initialValues } as SignatureIntegration)
-        : defaultValues;
+    const formInitialValues = merge({}, defaultValues, initialValues);
```

Comment:
Reviewing from bottom up.

Lack of type here is evidence that TypeScript infers it, which tip in Visual Studio Code confirms.

I will leave to your best judgement the pro and con to include type annotations as discussed in comment:
https://github.com/stackrox/stackrox/pull/12874/files#r1788126907

Reply:
added type annotations